smrf/distribute/solar.py

```diff
@@ -584,10 +494,44 @@ def calc_net(self, albedo_vis, albedo_ir):
         self.net_solar = vv_n + vir_n
         self.net_solar = utils.set_min_max(self.net_solar, self.min, self.max)
 
-    def calc_ir(self, min_storm_day, wy_day, tz_min_west, wyear, cosz, azimuth):
+    def calc_stoporad(self, date_time, illum_ang, cosz, azimuth,
+                      albedo_surface, wavelength_range=[0.28, 0.7]):
```

Comment:
Can't you use the constant from above here?
If not do something like
```python
def calc_stoporad(self, ...., wavelength_range=None)

if wavelength_range is None:
    wavelength_range = self. VISIBLE_WAVELENGTHS
```

Reply:
Added the two wavelength constants to the `constant` file just to keep everything grouped together.